src/test/java/org/sagebionetworks/bridge/services/StudyConsentServiceMockTest.java

```diff
@@ -1,34 +1,264 @@
 package org.sagebionetworks.bridge.services;
 
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static com.amazonaws.services.s3.model.CannedAccessControlList.PublicRead;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertSame;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.annotation.Nonnull;
 
 import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.amazonaws.services.s3.model.PutObjectRequest;
 import com.google.common.collect.ImmutableList;
 
+import org.apache.commons.io.IOUtils;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+import org.springframework.core.io.ByteArrayResource;
+import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
+import org.sagebionetworks.bridge.config.BridgeConfig;
 import org.sagebionetworks.bridge.dao.StudyConsentDao;
+import org.sagebionetworks.bridge.exceptions.BridgeServiceException;
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.models.studies.MimeType;
+import org.sagebionetworks.bridge.models.studies.Study;
 import org.sagebionetworks.bridge.models.subpopulations.StudyConsent;
+import org.sagebionetworks.bridge.models.subpopulations.StudyConsentForm;
+import org.sagebionetworks.bridge.models.subpopulations.StudyConsentView;
+import org.sagebionetworks.bridge.models.subpopulations.Subpopulation;
 import org.sagebionetworks.bridge.models.subpopulations.SubpopulationGuid;
+import org.sagebionetworks.bridge.s3.S3Helper;
+import org.sagebionetworks.bridge.validators.StudyConsentValidator;
 
-public class StudyConsentServiceMockTest {
+public class StudyConsentServiceMockTest extends Mockito {
```

Comment:
Do we have a test that's the equivalent of the following tests:

* invalidMarkupIsFixed()
* fullDocumentsAreConvertedToFragments()
* ckeditorMarkupIsPreserved()
* evenVeryBrokenContentIsFixed()

Reply:
No, I just left these out for no reason in particular. They are all added now.